python/hail/docs/annotationdb.rst

```diff
@@ -0,0 +1,234 @@
+.. _sec-annotationdb:
+
+===================
+Annotation Database
+===================
+
+This database contains a curated collection of variant annotations in Hail-friendly format, for use in Hail analysis pipelines. 
+
+.. _here: https://console.cloud.google.com/storage/browser/annotationdb/?project=broad-ctsa
```

Comment:
Yes, good catch. Had a link to the bucket in the description, but decided to remove it temporarily until we get a handle on the charges associated with people downloading large amounts of data from the bucket, and how we might get Google to charge the downloaders instead of us.